### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by devism** [Sommerregen/grav-theme-zsimplex#5]
After activating the theme I get a 404 page does not exist. Only the Home and Jobs links are showing and there is a popup saying it can't detect the language of the page. Also another popup says `Hyphenator.js cannot read property to lowerCaseOf Null`. I have all the required plugins for this theme already installed.

**Comment by Sommerregen** [comment]
Hi @devism ,

first of all make sure that line [default.html.twig#L2](https://github.com/Sommerregen/grav-theme-zsimplex/blob/develop/templates/default.html.twig#L2) has a language set. Second, which browser are you using? It might be that the hypenator (https://github.com/mnater/Hyphenator) makes some problems on your device.

**Comment by Sommerregen** [comment]
@devism If the aforementioned line has no language set, try

```twig
<html lang="{{ grav.language.getLanguage ?: (theme_config.default_lang|default('en')) }}">
```

**Comment by Sommerregen** [comment]
I also had some problems with latest Grav. I released a new version today **v2.2.1** that should fix this issue.
